runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSubscribeFactory.java

```diff
@@ -1455,6 +1461,7 @@ private void onKafkaData(
                         expireInterval = timestamp + helper.timeout - now().toEpochMilli();
                         if (expireInterval < 0)
                         {
+                            expiredMessage = true;
```

Comment:
```
long expireInterval;
if (helper.timeout != -1)
{
   ...
}
else
{
    expireInterval = helper.timeout;
}
```
can this be simplified to:
```
long expireInterval = -1;
if (helper.timeout != -1)
{
   ...
}
```

Reply:
Originally I wanted to write this like you showed. But as we're using the expireInterval in a lambda we'd get:
"Variable used in lambda expression should be final or effectively final".